.golangci.yml

```diff
@@ -0,0 +1,103 @@
+run:
+  timeout: 2m
+
+  # Skip the eth bindings.
+  skip-dirs:
+    - backend/ethereum/bindings/
+
+  # Do not change go.{mod,sum}.
+  modules-download-mode: readonly
+
+linters:
+  enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - dupl
+    - errcheck
+    - exportloopref
+    - funlen
+    # We use init() functions.
+    #- gochecknoinits
+    - goconst
+    - gocritic
+    - gocyclo
+    - gofmt
+    - goimports
+    # This is stupid since it reports every constant as magic-number.
+    #- gomnd
+    - goprintffuncname
+    - gosec
+    - gosimple
+    - govet
+    - ineffassign
+    # This thing reports long lines, but we have a lot of them…
+    #- lll
+    - misspell
+    # We often just have a 'return' when using named returns.
+    #- nakedret
+    - nolintlint
+    - rowserrcheck
+    - staticcheck
+    - structcheck
+    - stylecheck
+    - typecheck
+    - unconvert
+    - unparam
+    - unused
+    - varcheck
+    - whitespace
+    - asciicheck
+    # We often use globals for backend stuff
+    #- gochecknoglobals
+    - gocognit
+    - godot
+    # Reports TODO/BUG/FIXME
+    #- godox
+    # Keeps reporting 'do not define dynamic errors, use wrapped static errors instead'.
+    #- goerr113
+    # We don't care about alignment right now.
+    #- maligned
+    - nestif
+    - prealloc
+    # This always reports that tests should be Black Box.
+    #- testpackage
+    #- wsl
+
+issues:
+  exclude-rules:
+    - path: pkg/test
```

Comment:
adapt

Reply:
Looks like `stylecheck` is happy with the code, including it for now.  
Updated config will be done in https://github.com/hyperledger-labs/go-perun/issues/132

Reply:
Then just remove the exclusion rule

Reply:
I removed the `stylecheck` from the exclusion list, or what do you mean?  
Looks like this now:
```pre
 - path: test
      linters:
        # 'Error return value … not checked'
        - errcheck 
        - funlen
        # Tests do not need to prealloc everything.
        - prealloc
```  
@sebastianst